src/Command/Cache/DoctrineResultCacheWarmerCommand.php

```diff
@@ -69,26 +67,25 @@ class DoctrineResultCacheWarmerCommand extends Command
     /** @var int */
     private $batchSize;
 
-    /** @var array int[] */
-    private $userIds = [];
+    /** @var array string[] */
+    private $usernames = [];
 
     /** @var array */
```

Comment:
array string[]

Reply:
`string[]` is enough, `array` is redundant, fixed, thanks!